cpuinfo.go

```diff
@@ -190,15 +190,26 @@ func parseCPUInfoARM(info []byte) ([]CPUInfo, error) {
 	scanner := bufio.NewScanner(bytes.NewReader(info))
 
 	firstLine := firstNonEmptyLine(scanner)
-	if !strings.HasPrefix(firstLine, "Processor") || !strings.Contains(firstLine, ":") {
+	match, _ := regexp.MatchString("^[Pp]rocessor", firstLine)
+	if !match || !strings.Contains(firstLine, ":") {
 		return nil, errors.New("invalid cpuinfo file: " + firstLine)
 	}
 	field := strings.SplitN(firstLine, ": ", 2)
-	commonCPUInfo := CPUInfo{VendorID: field[1]}
-
 	cpuinfo := []CPUInfo{}
-	i := -1
 	featuresLine := ""
+	commonCPUInfo := CPUInfo{}
+	i := 0
+	if field[0] == "Processor " {
+		commonCPUInfo = CPUInfo{VendorID: field[1]}
```

Comment:
Vendor ID in the old format seems to become Model ID in the new. Sticking to one seems like a good idea, possibly Model as that's spelled out in the new format. Wdyt?

Reply:
I saw this change but my approach was trying to leave the old code as it was. As you say, I also prefer having one format instead of two. Let's use Model then.